index.html

```diff
@@ -203,11 +203,11 @@ <h4>About Me</h4>
      <section class="get-intouch">
          <h4>I'm always intrested in hearing about new projects, so 
              if you'd like to get chat please get in touch</h4>
-             <form>
-                 <input type="text" placeholder="Full name">
-                 <input type="email" placeholder="Email address">
-                 <textarea>Enter text here</textarea>
-                 <button type="button">Get in touch</button>
+             <form action="https://formspree.io/f/moqrdrwj" method="POST">
+                 <input type="text" name="name" maxlength="30" placeholder="Full name" required>
+                 <input type="email" name="email" placeholder="Email address" required>
+                 <textarea name="Message" type="text" maxlength="500" placeholder="Enter text here" required></textarea>
```

Comment:
* Please do not use the attribute "type." It is not allowed on element "textarea" at this point.

Reply:
Thanks @Daphineatim,
I am going to work on that